index.html

```diff
@@ -9,18 +9,64 @@
     <title>Document</title>
 </head>
 <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-dark sticky-top">
+        <div class="d-flex flex-grow-1">
+            <span class="w-100 d-lg-none d-block">
+                <!-- hidden spacer to center brand on mobile --></span>
+            <a class="navbar-brand d-none d-lg-inline-block text-white" href="#"> Book App </a>
+            <a class="navbar-brand-two mx-auto d-lg-none d-inline-block" href="#">
+                <img src="//placehold.it/40?text=LOGO" alt="logo">
+            </a>
+            <div class="w-100 text-right">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#myNavbar">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+        </div>
+        <div class="collapse navbar-collapse flex-grow-1 text-right" id="myNavbar">
+            <ul class="navbar-nav ms-auto flex-nowrap">
+               
+                <li class="nav-item" >
+                    <a href="#" id="list"  class="nav-link m-2 menu-item text-white">List</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" id="add-nav"  class="nav-link m-2 menu-item text-white">Add a book</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" id="contact-nav" class="nav-link m-2 menu-item text-white">Contact</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
```

Comment:
- [ ] The current date is expected to be displayed below the navigation bar, but we don't have that at the moment. For this requirement, you should use the [Luxon library.](https://moment.github.io/luxon/docs/manual/install.html) See [Get the current date and time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time)